docs/en/ingest-management/elastic-agent/configuration/autodiscovery/elastic-agent-kubernetes-autodiscovery.asciidoc

```diff
@@ -0,0 +1,10 @@
+[[elastic-agent-kubernetes-autodiscovery]]
+= Kubernetes autodiscovery with {agent}
+
+To use autodiscovery, you will need to modify the manifest file of the {agent}. Refer to <<running-on-kubernetes-standalone>> to learn how to access it.
+
```

Comment:
Team do you think we need 2 lines for Autodiscovery here?

Suggestion copied from here https://www.elastic.co/guide/en/beats/filebeat/current/configuration-autodiscover.html


When you run applications on containers, they become moving targets to the monitoring system. Autodiscover allows you to track them and adapt settings as changes happen. The autodiscover subsystem can monitor services as they start running.

Reply:
Yes as top comment. Above line 4 I would say